tests/test_cordex.py

```diff
@@ -0,0 +1,24 @@
+from STACpopulator.extensions.cordex6 import Cordex6DataModel
+
+
+def get_test_data():
+    import requests
+    from siphon.catalog import TDSCatalog
+    import xncml
+    from STACpopulator.stac_utils import numpy_to_python_datatypes
+
+    cat = TDSCatalog("https://pavics.ouranos.ca/twitcher/ows/proxy/thredds/catalog/birdhouse/disk2/ouranos/CORDEX/CMIP6/DD/NAM-12/OURANOS/MPI-ESM1-2-LR/ssp370/r1i1p1f1/CRCM5/v1-r1/day/tas/v20231208/catalog.html")
```

Comment:
Can we not use a real dataset that has to be remotely accessed in these tests? It would be preferable to mock the data so that these tests can be run offline and they won't be dependent on the PAVICS THREDDS server to pass.

Reply:
Agree, I'll make a static version.